types/jasmine/jasmine-tests.ts

```diff
@@ -500,6 +500,53 @@ describe("A spy, when configured to fake a series of return values", () => {
     });
 });
 
+describe("A spy, when configured to fake a promised return value", () => {
+    const bar = 10;
+    const foo = {
```

Comment:
Please use
```
const foo = jasmine.createSpyObj(['getAsyncBar']);
foo.getAsyncBar.and.resolveTo(bar);
```

Plus: I would prefer more useful names than foo and bar ;-)

Reply:
I might prefer useful names too, but we got a bunch of foo bar everywhere and I'd stick to the existing conventions